e2e/test/scenarios/dashboard-filters/dashboard-filters-auto-apply.cy.spec.js

```diff
@@ -69,17 +69,13 @@ describe("scenarios > dashboards > filters > auto apply", () => {
     cy.wait("@cardQuery");
 
     // changing parameter values by default should reload affected questions
-    filterWidget().within(() => {
-      cy.findByText(FILTER.name).click();
-    });
+    filterWidget().findByText(FILTER.name).click();
```

Comment:
Clean up unnecessary `within` that only contains a single action inside to use chaining syntax instead.